plans/views.py

```diff
@@ -401,6 +401,12 @@ def get_redirect_url(self, **kwargs):
         return redirect_url
 
 
+def billing_info_create_or_update(request, *args, **kwargs):
```

Comment:
Would be great to have some documentation and proper reasoning on this function.